cookbooks/aws-parallelcluster-common/resources/lustre/partial/_mount_unmount.rb

```diff
@@ -116,6 +116,11 @@ def initialize(node, resource, index)
         @mount_options = %w(defaults _netdev flock user_xattr noatime noauto x-systemd.automount)
         @device_name = "#{@dns_name}@tcp:/#{@mount_name}"
         @fstype = 'lustre'
+
+      elsif @type == 'FILECACHE'
+        @mount_options = %w(defaults _netdev flock user_xattr noatime noauto x-systemd.automount x-systemd.requires=network.service)
+        @device_name = "#{@dns_name}@tcp:/#{@mount_name}"
+        @fstype = 'lustre'
```

Comment:
Is file cache always a lustre filesystem?

Reply:
Yes